### PR TITLE
Show more button

### PR DESCRIPTION
**Pull request by katsuraku**

### CODE REVIEW

**Comment by FTgremlin** [comment]
y5d5m8d

File: src/shared/components/RealtimeArticleList.js [hunk 1]
```diff
@@ -143,13 +154,7 @@ export default class RealtimeArticleList extends React.Component {
             />
           </Col>
         </Row>
-        <Row>
-          <Col xs={12}>
-            <Button onClick={clickHandler}>
-              {buttText}
```
Comment (FTgremlin) [inline src/shared/components/RealtimeArticleList.js]: as in `buttonText`?

**Comment by katsuraku** [comment]
I am not the coiner of that const. But I think that is correct... it is as in buttonText

File: src/shared/components/RealtimeArticleList.js (hunk 1 above)
Comment (katsuraku) [inline src/shared/components/RealtimeArticleList.js]: I am not the coiner of that const. But I think that is correct... it is as in buttonText

**Comment by FTgremlin** [comment]
giphy-3

**Comment by radiodario** [comment]
you forgot to rebase and squash @FTgremlin

**Pull request merged**